doc/concepts/pipeline-concepts/datum/glob-pattern.md

```diff
@@ -0,0 +1,124 @@
+# Glob Pattern
+
+A glob pattern defines datums in your Pachyderm repository and how
+your data processing is spread among Pachyderm worker pods. A standalone
+datum can be a whole
+repository, each top filesystem object in the root folder,
+each filesystem object in a subdirectory, and so on. Each time
```

Comment:
Tough to read...a whole repository, the top filesystem objects in the root folder (or) the filesystem objects in a subdirectory, (if no "or" include etc. or an so on).